index.html

```diff
@@ -8,17 +8,18 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <h1>Awesome Books</h1>
+  <h1> All awesome Books</h1>
   <div class="awesomeList"></div>
   <div class="InputForm">
+    <h2>Add a new book.</h2>
     <input type="text" id="title" placeholder="Title"/>
     <br>
     <input type="text" id ="author"  placeholder="Author"/>
     <br>
-    <Button class="button" type="submit">Add Book</Button>
+    <Button class="button" type="submit">Add</Button>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** For good tags semantics  I recommend you change this `<Button>` tag to `<button>` to make your code base professional.